README.md

```diff
@@ -273,7 +273,7 @@ _Note: The `app.use(...)` call here is slightly different than what Express user
 
 ## TypeScript Types
 
-To get type defifinitions for this project:
```

Comment:
but isn't `defifi` more fun? 😂

Reply:
defifinitely, but I was worried if the README found out people were laughing at it behind its back, it could ruin its self-confidence.

I didn't think it was worth the risk of having to deal with a depressed & unconfident README.

Reply:
this made my day HAHAHAHAHAHAH